recipes/default.rb

```diff
@@ -165,17 +187,17 @@
 end
 
 # Register with kagent
-if node['kagent']['enabled'] == "true"
-  kagent_config service_name do
-    service "flyingduck"
-  end
+kagent_config service_name do
+  service "flyingduck"
+  restart_agent node["kagent"]["enabled"].casecmp?("true")
 end
 
 # Register with consul
 if service_discovery_enabled()
   # Register flyingduck with Consul
   consul_service "Registering Flyingduck with Consul" do
     service_definition "flyingduck.hcl.erb"
+    reload_consul node['install']['managed_docker_registry'].casecmp?("false")
```

Comment:
I'm a bit divided using this attribute to decide if we're running an Image for Cloud. Is it better if we check for the existence of `cloud/default` recipe?